### ISSUE TITLE
Improve search box in Graph view

### ISSUE THREAD

**Issue by benjello** [openfisca/legislation-explorer#12]
Two main concerns:
 - [x] wrong search on the name (class) of the variable. Exemple: Prime pour l'activité find ppa but ppa doesn't succeed to find ppa
 - [ ] Displaying the results of the search should be enhanced (larger display zone) to avoid horizonthal scroll bar as much as possible

This issue was randomly assigned. Feel free to change those ;-)

**Comment by pnbt** [comment]
Pushed the fix on branch graph, @cbenz  you just need to merge it :)

**Comment by pnbt** [comment]
By the way @cbenz  we don't see the variable name on the right side, I thought we had them?

**Comment by cbenz** [comment]
I can't deploy for now, it is blocked by https://github.com/openfisca/openfisca-web-api/pull/67 :)

**Comment by cbenz** [comment]
Deployed.

@pnbt But still scrollbars in the autocompleter.

**Comment by benjello** [comment]
May be using smaller fonts.
Would be nice to have  the label and the class name.
I know that is more text to handle but still it would be really nice.

**Comment by pnbt** [comment]
I pushed an update to do all of that in branch graph, see if you like it
@cbenz​
